pandapipes/test/topology/test_nxgraph.py

```diff
@@ -19,8 +20,18 @@ def test_include_branches():
 
     mg = top.create_nxgraph(net, include_pipes=True, include_valves=True, include_pumps=False)
     assert len(mg.nodes()) == len(net.junction)
-    assert len(mg.edges()) == len(net.pipe) + len(net.valve) 
+    assert len(mg.edges()) == len(net.pipe)
```

Comment:
I don't get, why the valves are not taken into account, even though "include_valves" is set to True.

Reply:
See my statement above. I may write:
assert len(mg.edges()) == len(net.pipe) + net.valve.in_service.sum()